cookbooks/aws-parallelcluster-slurm/templates/default/slurm/login_nodes_daemon_service/login_nodes_daemon_script.sh.erb

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
```

Comment:
*Minor*
It's a good practice to document the scope and usage of a script with an introductory comment

Reply:
Done!